CONTRIBUTION.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to Trace Viewer extension
+
+Thanks for your interest in the [Trace Viewer extension][trace-viewer]! The following is a set of 
+guidelines for contributing to the Trace Viewer extension for Theia applications.
+
+## How to Contribute
+
+⚠️ **Linux only!** Currently, the development environment for this project only works on Linux.
+
+In order to contribute, please first [open an issue][issues] that clearly describes the bug you 
+intend to fix or the feature you would like to add. Make sure you provide a way to reproduce the bug 
+or test the proposed feature. 
+
+Once you have your code ready for review, please  [open a pull request][pull-requests]. Please follow
+the [pull request guidelines][pr-guide]. If you are new to the project, the 
+[new contributors][new-contributors] section provides useful information to get you started. A 
+committer of the Trace Extension will then review your contribution and help to get it merged.
+
+Please note that before your pull request can be accepted, you must electronically sign the 
+[Eclipse Contributor Agreement][eca]. For more information, see the 
+[Eclipse Foundation Project Handbook][commiter-handbook].
+
+### Pull request guidelines 
+
+**Changes to the project** are made by submitting code with a pull request (PR).
+
+* [How to write and submit changes][creating-changes]
+* [Example pull request][issue-402]
+
+**Good commit messages** make it easier to review code and understand why the changes were made. 
+Please include a:
+
+* *Title:* Concise and complete title written in imperative (e.g. "Update Gitpod demo screenshots"
+or "Single-click to select or open trace")
+* *Problem:* What is the situation that needs to be resolved? Why does the problem need fixing? 
+Link to related issues (e.g. "Fixes [#317][issue-317]").
+* *Solution:* What changes were made to resolve the situation? Why are these changes the right fix?
+* *Impact:* What impact do these changes have? (e.g. Numbers to show a performance improvement, 
+screenshots or a video for a UI change)
+* [*Sign-off:*][sign-off] Use your full name and a long-term email address. This certifies that you 
+have written the code and that, from a licensing perspective, the code is appropriate for use in open-source.
+
+Other commit information:
+
+* [How to format the message][commit-message-message]
+* [Example commit message][commit-message-example]
+
+### New contributors
```

Comment:
Since the other chapters are moved up, have this chapter as heading 2: ## New contributors